lib/libutils/ext/trace.c

```diff
@@ -133,6 +149,12 @@ void trace_vprintf(const char *function, int line, int level, bool level_ok,
 	boffs += res;
 
 	if (level_ok && (BIT(level) & CFG_MSG_LONG_PREFIX_MASK)) {
+		/* Print the current guest ID or 0 for Nexus */
+		res = print_guest_id(buf + boffs, sizeof(buf) - boffs);
+		if (res < 0)
```

Comment:
I think `scripts/symbolize.py` needs updating (`STACK_ADDR_RE`).

Reply:
Added a commit for this, let me know if you'd rather have it in a single commit.

Reply:
Thanks, one commit would be better indeed. With that:

`Reviewed-by: Jerome Forissier <jerome.forissier@linaro.org>`